runtime/src/main/java/com/flipkart/flux/controller/FSMVisualizationController.java

```diff
@@ -39,19 +39,13 @@
 @Controller
 public class FSMVisualizationController {
 
-    /* Guice injector which provides flux configuration using com.flipkart.flux.guice.module.ConfigModule */
-    private Injector configInjector;
 
-    /* Constructor*/
-    public FSMVisualizationController(Injector configInjector) {
-        this.configInjector = configInjector;
-    }
 
     @RequestMapping(value = {"/fsmview"}, method = RequestMethod.GET)
     public String fsmview(@RequestParam(value = "fsmid", required = false) String fsmId, ModelMap modelMap, HttpServletRequest request) throws UnknownHostException {
-        int fluxApiPort = configInjector.getInstance(Key.get(Integer.class, Names.named("Api.service.port")));
+//        int fluxApiPort = configInjector.getInstance(Key.get(Integer.class, Names.named("Api.service.port")));
         String fluxApiHost = InetAddress.getLocalHost().getHostAddress();
-        String fluxApiUrl = System.getProperty("flux.runtimeUrl")== null ? "http://" + fluxApiHost   + ":" + fluxApiPort : System.getProperty("flux.runtimeUrl");
+        String fluxApiUrl = System.getProperty("flux.runtimeUrl")== null ? "http://" + fluxApiHost   + ":" + 9998 : System.getProperty("flux.runtimeUrl");
```

Comment:
cause the dashboard server and api server run on different port, and dashboard server makes a call to api server to get fsm related details. will see if this can be refactored better.